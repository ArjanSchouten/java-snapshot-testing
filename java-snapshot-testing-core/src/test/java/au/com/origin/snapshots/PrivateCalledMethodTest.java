package au.com.origin.snapshots;

import au.com.origin.snapshots.config.BaseSnapshotConfig;
import org.junit.jupiter.api.AfterAll;
import org.junit.jupiter.api.BeforeAll;
import org.junit.jupiter.api.Test;

import static au.com.origin.snapshots.SnapshotMatcher.expect;

class PrivateCalledMethodTest {

  @BeforeAll
  static void beforeAll() {
    SnapshotMatcher.start(new BaseSnapshotConfig());
  }

  @AfterAll
  static void afterAll() {
    SnapshotMatcher.validateSnapshots();
  }

  @Test
  void testName() {
    testBasedOnArgs("testContent");
  }

  private void testBasedOnArgs(String arg) {
    expect(arg).toMatchSnapshot();
  }
}
