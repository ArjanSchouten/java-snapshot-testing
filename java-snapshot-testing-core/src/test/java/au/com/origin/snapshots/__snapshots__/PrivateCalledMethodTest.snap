au.com.origin.snapshots.PrivateCalledMethodTest.beforeAll=[
  "testContent"
]


au.com.origin.snapshots.PrivateCalledMethodTest.testBasedOnArgs=[
  "testContent"
]


au.com.origin.snapshots.PrivateCalledMethodTest.testName=[
  "testContent"
]